docs/modules/IPFS.md

```diff
@@ -26,11 +26,11 @@ v0.0.1
 **`Example`**
 
 ```ts
-validateIpfsUrl('ipfs://{hash}'); //=> 'https://2eff.lukso.dev/ipfs/{hash}'
-validateIpfsUrl('https://google.com/something'); //=> 'https://google.com/something'
-validateIpfsUrl(''); //=> ''
+validateIpfsUrl('ipfs://{hash}') => 'https://2eff.lukso.dev/ipfs/{hash}'
```

Comment:
Suggest keeping as before as it is a comment here.
If someone copy paste the code snippet, it will not run